fastlane/ScreenshotFastfile

```diff
@@ -34,7 +34,7 @@ platform :ios do
       derived_data_path: derived_data_path,
     )
 
-    languages = "ru".split(" ")
+    languages = "da de-DE en-AU en-CA en-GB es-ES fr-FR id it ja ko no nl-NL pt-BR pt-PT ru sv th tr zh-Hans zh-Hant en-US".split(" ")
```

Comment:
Random question: have you considered making this an array? Would it help maintaining the setup? 🤔

Reply:
Yeah, good question – ideally we'd pull this from a global "list of locales we support" that's aligned between screenshots, metadata, glotpress, etc. For now I'm inclined to leave it, but I look forwarding to having it be a lot nicer, as you say!